From 8081928da2083c9a61d56b4cf337d16c584a9adb Mon Sep 17 00:00:00 2001
From: "Radu Pirea (NXP OSS)" <radu-nicolae.pirea@oss.nxp.com>
Date: Tue, 8 Feb 2022 16:10:22 +0200
Subject: [PATCH 02/51] s32: sgmii: remove SGMII_MIN_SOC_REV_SUPPORTED

Remove SGMII_MIN_SOC_REV_SUPPORTED because is not definded.
The code guarded by SGMII_MIN_SOC_REV_SUPPORTED is not even compiled.

Issue: ALB-8367

Upstream-Status: Pending 

Signed-off-by: Radu Pirea (NXP OSS) <radu-nicolae.pirea@oss.nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/cpu/armv8/s32/s32-gen1/sgmii/sgmii.c | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/arch/arm/cpu/armv8/s32/s32-gen1/sgmii/sgmii.c b/arch/arm/cpu/armv8/s32/s32-gen1/sgmii/sgmii.c
index 71a7c919bf..3920d8393f 100644
--- a/arch/arm/cpu/armv8/s32/s32-gen1/sgmii/sgmii.c
+++ b/arch/arm/cpu/armv8/s32/s32-gen1/sgmii/sgmii.c
@@ -208,25 +208,6 @@ int s32_eth_xpcs_init(void __iomem *serdes_base, int platform_serdes_id,
 		      enum serdes_clock clktype,
 		      enum serdes_clock_fmhz fmhz)
 {
-#ifdef SGMII_MIN_SOC_REV_SUPPORTED
-	u32 raw_rev = 0;
-
-	/* construct a revision number based on major, minor and subminor,
-	 * each part using one hex digit
-	 */
-	raw_rev = (get_siul2_midr1_major() << 8) |
-		  (get_siul2_midr1_minor() << 4) |
-		  (get_siul2_midr2_subminor());
-
-	if (raw_rev < SGMII_MIN_SOC_REV_SUPPORTED) {
-		printf("SGMII not supported on rev.");
-		printf("%d.%d.%d\n", get_siul2_midr1_major() + 1,
-		       get_siul2_midr1_minor(),
-		       get_siul2_midr2_subminor());
-		return -ENXIO;
-	}
-#endif /* SGMII_MIN_SOC_REV_SUPPORTED */
-
 	struct s32_xpcs_cfg *serdes = s32_get_serdes_priv(platform_serdes_id);
 	int ret = 0;
 	u32 init_flags = 0;
-- 
2.17.1

